fleet_vehicle_capacity/models/fleet_vehicle.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 - TODAY, Marcel Savegnago <marcel.savegnago@escodoo.com.br>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class FleetVehicle(models.Model):
+
+    _inherit = "fleet.vehicle"
+
+    vehicle_weight = fields.Float(
+        related="model_id.vehicle_weight",
+        string="Vehicle Weight",
+        help="The weight of the vehicle.",
+        readonly=False,
+    )
+
+    weight_capacity = fields.Float(
+        related="model_id.weight_capacity",
+        string="Weight Capacity",
+        help="The maximum weight capacity the vehicle can support.",
+        readonly=False,
+    )
+
+    volume_capacity = fields.Float(
+        related="model_id.volume_capacity",
+        string="Volume Capacity",
+        help="The maximum volume capacity the vehicle can support.",
+        readonly=False,
+    )
+
+    passenger_capacity = fields.Float(
+        related="model_id.passenger_capacity",
+        string="Passenger Capacity",
+        help="The maximum passenger capacity the vehicle can support.",
+        readonly=False,
```

Comment:
It feels prone to error to have `readonly=False` here.

Let's say I have a fleet with many vehicles, they're all the same model.
For one of the vehicles, I've a modified/disabled seat, so I'm tempted to decrease this passenger capacity.
I'll be unknowingly modifying the model, and all the other vehicles too, wouldn't I?

I'd then go for any of the following two alternatives:

1) Have these fields be computed from the model with `@api.depends('model_id')` (achieving the onchange+default effect). Then, users can modify these values on a per vehicle basis.

2) Have these related fields as readonly, and so it encourage users to modify the model instead